open_humans/member_views.py

```diff
@@ -87,8 +88,8 @@ def get_queryset(self):
             if not badge_exists:
                 raise Http404()
 
-            queryset = queryset.filter(
-                badges__contains=[{'label': filter_name}])
+            project = id_label_to_project(filter_name)
+            queryset = queryset.filter(datarequestproject=project)
```

Comment:
oops! It looks to me like this ends up returning the `Member` that is `project.coordinator`?

Maybe you want something like `datarequestprojectmember__project=project` but ... I think also need to check that they're authorized/joined.

Reply:
o, oops